### PR TITLE
Deployment: `BalancerPoolDataQueries`

### PR DESCRIPTION
**Pull request by jubeira**
# Deployment: `BalancerPoolDataQueries`

## Deployment tasks checklist:

- [x] Contract(s) deployed to all target networks / testnets
- [x] TX IDs are properly generated
- N/A Action IDs are generated after deployment (if applicable) 
- [x] Outputs for all target networks are generated, and linked in the task `readme`
- [x] Addresses files are up to date with the new deployment 
- [x] Contracts are verified in every network 
- [x] Deployments `CHANGELOG` is updated
- [x] There are no code changes of any kind

## Issue Resolution

N/A

### CODE REVIEW

**Review by EndymionJkb — APPROVED** [review]
One missing link - otherwise LGTM. Checked that it's verified in every network.

File: tasks/20230613-balancer-pool-data-queries/readme.md [hunk 1]
```diff
@@ -2,4 +2,13 @@
 
 Deployment of the `BalancerPoolDataQueries` contract. It provides a way to perform bulk queries that is useful for optimizing SOR initialization of on chain data.
 
+- [Ethereum mainnet addresses](./output/mainnet.json)
+- [Polygon mainnet addresses](./output/polygon.json)
+- [Arbitrum mainnet addresses](./output/arbitrum.json)
+- [Optimism mainnet addresses](./output/optimism.json)
```
[the remark this answers is not in the document]
Comment (jubeira) [reply tasks/20230613-balancer-pool-data-queries/readme.md:8]: good catch, thanks!

**Comment by jubeira** [comment]
well, it seems that committing the suggestion also makes the review stale :(

**Review by EndymionJkb — APPROVED** [review]

**Comment by EndymionJkb** [comment]
> well, it seems that committing the suggestion also makes the review stale :(

I guess technically it should; we were spoiled by the monorepo's policy of preserving approvals no matter what. Maybe it should only dismiss if there's a completely new commit, vs. just adopting a suggestion. Not sure if that's possible.

**Pull request merged**